package.json

```diff
@@ -36,10 +36,10 @@
   },
   "private": true,
   "dependencies": {
-    "@alfresco/adf-content-services": "3.2.0-beta7",
-    "@alfresco/adf-core": "3.2.0-beta7",
-    "@alfresco/adf-extensions": "3.2.0-beta7",
-    "@alfresco/js-api": "3.2.0-beta7",
+    "@alfresco/adf-content-services": "3.2.0-90276dd4988b80cb0e4b6912b41e8d0c810a313c",
+    "@alfresco/adf-core": "3.2.0-90276dd4988b80cb0e4b6912b41e8d0c810a313c",
+    "@alfresco/adf-extensions": "3.2.0-90276dd4988b80cb0e4b6912b41e8d0c810a313c",
+    "@alfresco/js-api": "~3.2.0-fa5916ff413131513c3e382d7f27dd9b4cfa0e7e",
```

Comment:
needs exact version. You can upgrade to the latest alpha with the following command:

```
./scripts/update-version.sh -v alpha
```